src/ServiceControl.Config.Tests/AutoAttachmentModuleTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace ServiceControl.Config.Tests
+{
+    using Autofac;
+    using Caliburn.Micro;
+    using Framework;
+    using Framework.Modules;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class AutoAttachmentModuleTests
+    {
+        [Test]
+        public void AttachmentsAreAttached()
+        {
+            AssemblySource.Instance.Add(typeof(AutoAttachmentModuleTests).Assembly);
```

Comment:
This being static, should it be done in a pre-test step? (and potentially cleaned up after?)

Reply:
Good call. Moved this to setup method and added teardown method to remove it afterwards